A3A/addons/core/functions/CREATE/fn_attackHQ.sqf

```diff
@@ -20,7 +20,7 @@ params ["_side", "_airbase", "_delay"];			// Side is now specified
 private _targPos = markerPos "Synd_HQ";
 private _faction = Faction(_side);
 
-bigAttackInProgress = true; publicVariable "bigAttackInProgress";
+//bigAttackInProgress = true; publicVariable "bigAttackInProgress";
```

Comment:
Surely a HQ attack is a big attack?

Reply:
This comment should be removed given it dictates if extra missions are created during already laggy events